scanners/pshtt.py

```diff
@@ -50,7 +50,7 @@ def scan(domain, options):
 
     if (force is False) and (os.path.exists(cache_pshtt)):
         logging.debug("\tCached.")
-        raw = open(cache_pshtt).read()
+        raw = utils.read(cache_pshtt)
```

Comment:
Explicitly closing some not-explicitly-closed file handles, which Docker was yelling about. In a couple places below as well.